src/importedentity.cpp

```diff
@@ -79,7 +79,7 @@ bool ImportedEntity::doEquals(const ImportedEntityPtr &other) const
         && mPimpl->mImportReference == other->importReference()) {
         return mPimpl->mImportSource->equals(other->importSource());
     }
-    return importMatches;
+    return importMatches && mPimpl->mImportReference == other->importReference();
```

Comment:
Replace with `return importMatches && (mPimpl->mImportReference == other->importReference());` to avoid ambiguity. I believe we folllow that convention?

Reply:
[6fef492](https://github.com/cellml/libcellml/pull/1046/commits/6fef492eaa5d47e5b104695e12182acd0995d208)